ci/docker_cache.py

```diff
@@ -240,16 +234,36 @@ def script_name() -> str:
     logging.basicConfig(format='{}: %(asctime)-15s %(message)s'.format(script_name()))
 
     parser = argparse.ArgumentParser(description="Utility for preserving and loading Docker cache", epilog="")
+
     parser.add_argument("--docker-registry",
                         help="Docker hub registry name",
-                        type=str,
-                        required=True)
+                        default="mxnetci",
+                        type=str)
+
+    parser.add_argument("--docker-secret-name",
+                        help="secrets manager name",
+                        default="prod/DockerCache/DockerHubCredentials",
```

Comment:
The environment variables were chosen because they are environment specific variables (think about prod and test config).